aws-ecs/01-variables.tf

```
@@ -78,10 +91,16 @@ variable "rds_port" {
   default     = "5432"
 }
 
+variable "rds_backup_retention_period" {
+  description = "Pvault RDS backup_retention_period"
```

```suggestion
  description = "Vault RDS backup retention period (in minutes)"
```

If not minutes then change to the correct unit

---

I used the module syntax